.github/workflows/node.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  Tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Install
+        run: npm ci
+      - name: Test
+        run: npm run test:coverage
+        if: ${{ always() }}
+      - name: report coverage
+        uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
It would be nice to also to run the build

Reply:
I left the build out of the Github Actions because we're already building an deploying it in Vercel